views/contacts.ejs

```diff
@@ -5,28 +5,38 @@
 	<!-- Load bootstrap and fontawesome -->
 	<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.2/css/bootstrap.min.css">
 	<link rel="stylesheet" href="//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.min.css">
+	<!-- css -->
+	<link rel="stylesheet" href="/styles.css">
 
-	<style>
-
-	</style>
 </head>
 <body>
```

Comment:
Definitely haha. Sorry this app is a mess atm. I actually went though a whole TreeHouse course on authentication, cookies, and sessions yesterday, so I'm going to refactor what I've already written. I really dislike using the <%= syntax that ejs provides. That is really difficult to read! I wish it was `{{ code }}`. I might mess around with pug.